app/models/user.rb

```diff
@@ -1,2 +1,12 @@
 class User < ActiveRecord::Base
+  def self.find_or_create_from_auth(auth)
```

Comment:
Assignment Branch Condition size for find_or_create_from_auth is too high. [15.03/15]